packages/kit/src/views/Home/pages/HomePageView.tsx

```
@@ -265,12 +263,12 @@ export function HomePageView({
           networkId={network?.id ?? ''}
           accountId={account?.id ?? ''}
         >
-          <>{renderTabs()}</>
+          <>{tabs}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)